test/e2e/specs/cache.js

```diff
@@ -38,7 +38,7 @@ describe('Backendless.Cache', function() {
 
   it('contains', function() {
     return Promise.resolve()
-      .then(() => Backendless.Persistence.of(Foo).save(cacheValues.customType))
+      .then(() => Backendless.Data.of(Foo).save(cacheValues.customType))
```

Comment:
do not forget to add e2e test for GEO_Data

Reply:
Done